src/ansys/fluent/core/filereader/casereader.py

```diff
@@ -73,36 +76,53 @@ class CaseReader:
         Get the precision (1 or 2 for 1D of 2D)
     """
 
-    def __init__(self, hdf5_case_filepath: str):
+    def __init__(self, case_filepath: str):
+
         try:
-            file = h5py.File(hdf5_case_filepath)
+            if Path(case_filepath).suffix == ".h5":
```

Comment:
Like below, you could check for `cas.h5` right?

Reply:
Done :)